spaceship/lib/spaceship/tunes/member.rb

```diff
@@ -56,10 +56,11 @@ def preferred_currency
 
       def selected_apps
         parsed_apps = []
+        all_apps = Application.all
         raw_data["userSoftwares"]["value"]["grantedSoftwareAdamIds"].each do |app_id|
-          parsed_apps << Application.find(app_id)
+          parsed_apps << all_apps.select { |app| !app.apple_id.nil? && app.apple_id == app_id }
```

Comment:
```suggestion
          parsed_apps << all_apps.select { |app| app.apple_id == app_id }
```
You probably don't need to check for `nil` here ^ nil will never equal `app_id` so this will clean it up a bit